pastastore/plotting.py

```diff
@@ -715,6 +715,78 @@ def model(self, ml, label=True,
 
         return ax
 
+    def stresslinks(self, kinds=None, model_subnames=None, color_lines=False,
+                    alpha=0.3, figsize=(10, 8), legend=True, labels=False):
+        """Create a map for (a selection of) models with their accompanied
+        kinds of stresses to plot. 
+
+        Parameters
+        ----------
+            kinds: list, optional
+                Kinds of stresses to plot. Defaults to None.
+            model_subnames: list, optional
```

Comment:
Would just call this model_names and explain in the docstring partial matches are supported.